### DISCUSSION TITLE
Daily 23/10

### DISCUSSION THREAD

**Discussion by NataliaAlvarezIspc** [grupo-ispc-2023-2-0/workflix-app#260] [Daily]
- ¿Qué hice ayer?
- ¿Qué hice o haré hoy?
- Impedimentos o dificultades

**Comment by BraianTroncoso** [comment]
El sábado estuve realizando algunas de mis tareas asignadas. Contribuí ayudando a través de meet para poder solucionar algunos bugs que tenian. Ayer el domingo, también estuve realizando algunas tareas de la cual una no pude finalilzar, la de ocultar perfil dashboard, etc. Eran las 6 de la mña y seguia renegando para consumir un dato json que en teoria llega bien del back, pero por alguna razón justo el atributo que me hace falta no llega, no se, hoy voy a verlo con mas tranquilidad y avanzar en las que tengo asignadas para terminar el proyecto.
Se me complica la falta de tiempo al realizar muchas tareas diarias sin excepción de descanso de lun a lun. Me gustaria que se termine todo el miercoles, un par de dias antes, asi podemos ver con detenimiento por si hay algun bugs, no me gusta dejar todo para el último. Nos habiamos comprometidos en finalizar el sprint al menos en código para el viernes pasado, lo cual no fué asi por diversos problemas ya mencionados. Espero que todos pongamos de nuestra parte para terminarlo con dias de anticipacion y no el último dia, no deberia ser asi.

**Comment by NataliaAlvarezIspc** [comment]
El viernes, finalmente, no pude hacer nada de código porque nada me funcionaba. Estuve viendo un poco la documentación y cambiando algunas cosas. El sábado participé en el encuentro semanal del grupo, vimos que cosas nos están faltando. También estuve tratando de ayudar a algunos compañeros con tareas diversas. Luego estuve trabajando en las tareas que tenía asignadas. Pude terminar la de perfil para que traiga los datos del profesional logueado y otras tareas de front (agregue borde a los campos que se completan en login y registro, quité algunos bug que tenían los botones de las activitys de perfil, etc)
Hoy estuve acomodando casi todas las activitis para que sean responsive (perfilx2, restablecer, login, contacto y home) ya está todo subido al repo. Ahora voy a intentar hacer las tareas de dashboard de administrador y luego si me da el tiempo veré que pasa con el guardado de datos en registración.
Dificultades, como siempre, los pocos recursos de mi computadora que hacen todo mas lento y a medida que avanza el proyecto, más aún.

**Comment by MarielaGomez** [comment]
El Sábado nos reunimos mediante meet con algunos compañeros para pasar en limpio las tareas que faltaban y también revisar documentación, durante el mismo día y parte del Domingo me dediqué a terminar la tarea que me había asignado del formulario, hoy subí el archivo png de los casos de uso con el fondo blanco, continuaré a repasar el proyecto para ver que falta por seguir mejorando.

**Comment by Valentino399** [comment]
El sabado me junte con mis compañeros para hablar. Hoy estuve arreglando el código que hice que es subir un video en acerca de la app, me geneo error en lo que sería de java pude corregir eso, ahora tengo un problema y es que el video no me lo reconoce android studio no me deja registrar el archivo y voy a solucionar esto
